dotnet/src/Microsoft.AutoGen/RuntimeGateway.Grpc/Abstractions/IGateway.cs

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // IGateway.cs
+using Grpc.Core;
 using Microsoft.AutoGen.Contracts;
+using Microsoft.AutoGen.Protobuf;
 
-namespace Microsoft.AutoGen.Runtime.Grpc.Abstractions;
+namespace Microsoft.AutoGen.RuntimeGateway.Grpc.Abstractions;
 
 public interface IGateway : IGrainObserver
 {
     ValueTask<RpcResponse> InvokeRequestAsync(RpcRequest request);
     ValueTask BroadcastEventAsync(CloudEvent evt);
```

Comment:
Why is there both BroadcastEventAsync and WriteResponseAsync that take cloud events?

Reply:
broadcast was a holdover - allows sending a message to all workers. not needed.